README.md

```diff
@@ -233,18 +232,19 @@ Both fluent and functional syntax may be mixed and even pipe can be used
 fluently.
 
 ```python
+from expression.collections import seq, Seq
 xs = Seq.of(1, 2, 3).pipe(seq.map(...))
 ```
 
-### Options
+### Option
```

Comment:
I changed these headers to be the name of the type, rather than the type name pluralized